docs/migration.md

```diff
@@ -1204,7 +1214,131 @@ Docker Logs Source is not supported by the OpenTelemetry Collector.
+    ## Map Docker container environmental variables to resource attributes
+    env_vars_to_metric_labels:
+      BIN_DIR: bin-directory
+
+    ## Enable or disable particular metrics.
+    ## Full list of metrics with their default config is available at https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.73.0/receiver/dockerstatsreceiver/documentation.md
+    metrics:
+      container.cpu.usage.percpu:
+        enabled: true
+      container.network.io.usage.tx_dropped:
+        enabled: false
+```
+
+#### Name
+
+Please refer to [the Name section of Common configuration](#name-1).
+
+#### Description
+
+Please refer to [the Description section of Common configuration](#description-1).
+
+#### URI
+
+To specify URI, use `endpoint` option:
+
+```yaml
+receivers:
+  dockerstats:
+    endpoint: http://example.com
+```
+
+`Cert Path` option is not supported in OpenTelemetry Collector.
+
+#### Container filters
+
+Unfortunately, containers cannot be filtered by their name in OpenTelemetry Collector. The only way to filter containers is to exclude them by their image name through `excluded_images` option:
+
+```yaml
+receivers:
+  dockerstats/1:
+    excluded_images:
+      ## Exclude particular image
+      - docker.io/library/nginx:1.2
+      ## Exclude by regex
+      ## Note: regex must be but between / characters
+      - /other-docker-registry.*/
+      ## Exclude by glob
+      - exclude-*-this
+
+  dockerstats/2:
+    excluded_images:
+      ## Use negation: scrape metrics only from nginx containers
+      - !*nginx*
+      ## Negation for regexes requires using ! before the slash character
+      - !/.*nginx.*/
+```
+
+Valid names are strings, [regexes](https://pkg.go.dev/regexp) and [globs](https://github.com/gobwas/glob).
+Negation can also be used, for example glob `!my*container` will exclude all containers for which the image name doesn't match glob `my*container`.
+
+#### Source Host
+
+Please refer to [the Source Host section of Common configuration](#source-host).
+
+#### Source Category
+
+Please refer to [the Source Category section of Common configuration](#source-category).
+
+#### Fields
+
+Please refer to [the Fields section of Common configuration](#fields-1).
+
+#### Scan interval
+
+To indicate a scan interval, use `collection_interval` option:
+
+```yaml
+receivers:
+  dockerstats:
+    ## default is 10s
+    collection_interval: 20s
+```
+
+#### Metrics
+
+In OpenTelemetry Collector, some metrics are being emitted by default, meanwhile some have to be enabled. You can enable emitting a metric by setting `metrics.<metric_name>.enabled` to `true` and disable it by setting this option to `false`, for example:
+
+```yaml
+receivers:
+  dockerstats:
+    metrics:
+      container.cpu.usage.percpu:
+        enabled: true
+      container.network.io.usage.tx_dropped:
+        enabled: false
+```
+
+Full list of metrics available in this receiver can be found [here][dockerstatsmetrics].
```

Comment:
Could we reseatch what are installed collector metrics and how do they relate to OpenTelemetry Collector ones? It can be done in different PR

Reply:
I wanted to do that, but there are quite a lot:
https://help.sumologic.com/docs/send-data/installed-collectors/sources/docker-sources/#docker-metrics-definitions

do we have a list of some most commonly used?

Reply:
I think we should document all of them. Maybe create separate issue for this one

Reply:
the issue: https://jira.kumoroku.com/jira/browse/SUMO-215176